packages/gatsby-source-filesystem/src/__tests__/create-file-node.js

```diff
@@ -2,20 +2,23 @@ const path = require(`path`)
 
 const { createFileNode } = require(`../create-file-node`)
 
-describe(`create-file-node`, () => {
-  it(`creates a file node`, done => {
-    createFileNode(
-      path.resolve(`${__dirname}/fixtures/file.json`),
-      {},
-      (err, fileNode) => {
```

Comment:
Oh right :-)

Reply:
we need like a schema based testing thing for this i think. like `matchesSortOfThis()` :P

Reply:
`matchesSnapshotWithinLevenshteinDistanceOf(10)`